terraso_backend/apps/core/management/commands/loadbackup.py

```diff
@@ -0,0 +1,279 @@
+            raise CommandError(msg)
+
+        def cleanup():
+            if options["s3"]:
+                for path in data, migrations_file:
+                    os.unlink(path)
+
+        with connection.cursor() as cursor:
+            try:
+                # background task progress is saved in a DB table
+                # need to make sure this table is not deleted. otherwise things will break
+                cursor.execute(
+                    "SELECT tablename FROM pg_tables WHERE schemaname = 'public' "
+                    "AND tablename NOT IN ('core_backgroundtask', 'django_session');"
+                )
+                tables_to_drop = cursor.fetchall()
+                if session_pk:
+                    cursor.execute(
+                        "DELETE FROM django_session WHERE session_key != %s", (str(session_pk),)
+                    )
+                else:
+                    cursor.execute("DELETE FROM django_session")
+                for (table,) in tables_to_drop:
+                    cursor.execute(
+                        sql.SQL("DROP TABLE IF EXISTS {} CASCADE").format(sql.Identifier(table))
+                    )
+            except Exception:
+                msg = "Command failed resetting database"
+                logger.exception(msg)
+                connection.rollback()
+                cleanup()
+                raise CommandError(msg)
+
+            connection.commit()
+
+        try:
+            for app_label, version in migrations.items():
+                if app_label == "core":
+                    # need to skip the migration that creates the background tasks
+                    for version in self.CORE_MIGRATIONS_TO_SKIP:
+                        management.call_command(
+                            "migrate", "core", "{0:0>4}".format(version - 1), verbosity=0
+                        )
+                        management.call_command(
+                            "migrate", "core", "{0:0>4}".format(version), fake=True, verbosity=0
+                        )
+                    management.call_command("migrate", "core", verbosity=0)
+                kwargs = {}
+                if app_label == "sessions":
+                    kwargs["fake_initial"] = True
+                management.call_command("migrate", app_label, verbosity=0, **kwargs)
+
+            management.call_command(
+                "loaddata",
+                str(data.resolve()) if isinstance(data, Path) else data,
+                exclude=[
+                    "core.BackgroundTask",
+                    "contenttypes.contenttype",
+                    "auth.Permission",
+                    "sessions.Session",
+                    "admin.LogEntry",
+                ],
+            )
+
+            if user:
+                try:
+                    new_user = User.objects.get(email=user.email)
+                    # provide a fake "email" not in the database
+                    user.email = str(user.id)
```

Comment:
issue: please write a detailed comment explaining what is going on with the user ids, which tables are skipped, etc.